### ISSUE TITLE
Prompt/delay characters are not stripped from the middle of a message

### ISSUE THREAD

**Issue by Herschel** [onimitch/ffxi-balloon-ashitav4#3]
Prompt characters are not stripped from the middle of the message, causing extraneous characters to appear:

![Hersch_2024 01 25_230652](https://github.com/onimitch/ffxi-balloon-ashitav4/assets/36278/0d78b3c6-ee94-4825-8dbf-a0f4ccb7c373)

Test case:
```
tests.delay = {
    'Lathuya',
    'Now the crafting fee for the \x1e\x02magus shalwar\x1e\x01...\x7f\x36\x02\x07We require an up-front payment of 2 \x1e\x02Imperial mythril pieces\x1e\x01.\x7f\x31',
    ''
}
```

**Comment by onimitch** [comment]
Thanks for the bug report. I've had this one fixed for a little while (it was a regression in the previous release I think) but been a bit busy to finish testing and put out a new release. Will get it out asap

**Comment by onimitch** [comment]
Fixed in [Version 4.0.4](https://github.com/onimitch/ffxi-balloon-ashitav4/releases/tag/4.0.4)
